src/cljs/coffee_re_frame/engine.cljs

```diff
@@ -40,6 +41,9 @@
  ::next-step
  handle-next-step)
 
+(defn should-vibrate [remaining-seconds-in-step]
+  (<= remaining-seconds-in-step 3))
```

Comment:
This might need to be `4` not `3`? It seems to be vibrating one 2... 1... 30 on the next step. At least, from a visual standpoint

Reply:
For non-timed steps, i.e., when you get to `stir + swirl`, it just keeps vibrating since the timer is at 0